hermes_core/timedata.py

```diff
@@ -532,13 +636,19 @@ def add_measurement(self, measure_name: str, data: u.Quantity, meta: dict = None
                 f"Column '{measure_name}' must be a one-dimensional measurement. Split additional dimensions into unique measurenents."
             )
 
-        self._timeseries[measure_name] = data
+        self._timeseries[hermes_core.DEFAULT_TIMESERIES_KEY][measure_name] = data
```

Comment:
check which timeseries the measurement belongs in. also create an `add_timeseries`function.